apps/src/lib/util/PubSubService.js

```diff
@@ -47,7 +48,10 @@ import {PusherChannel, NullChannel} from './PubSubChannel';
  */
 export default function create(pubSubConfig) {
   if (pubSubConfig.usePusher) {
-    return new PusherService(pubSubConfig.pusherApplicationKey);
+    return new PusherService(
+      pubSubConfig.pusherApplicationKey,
+      _.pick(pubSubConfig, ['auth', 'authEndpoint', 'forceTLS'])
```

Comment:
Presence channels are a subset of [private channels](https://pusher.com/docs/channels/using_channels/private-channels), so clients must [authenticate with an auth endpoint](https://pusher.com/docs/channels/server_api/authenticating-users) on our servers.  We're also using Rails which has CSRF protection enabled by default, so we have to [pass as CSRF token to the auth endpoint](https://pusher.com/docs/channels/server_api/authenticating-users#csrf-protected-authentication-endpoints).